### PR TITLE
[WIP] Working on about page

### PR DESCRIPTION
**Pull request by roshancvp**
* Trying to link the about page at "..lab/about.html"
* Then link the page inside an iframe inside about/plugin.ts

### CODE REVIEW

File: jupyterlab/__init__.py [hunk 1]
```diff
@@ -31,12 +32,18 @@ def get(self):
     def get_template(self, name):
         return FILE_LOADER.load(self.settings['jinja2_env'], name)
 
+class AboutHandler(AuthenticatedHandler):
+
```
Comment (willingc) [inline jupyterlab/__init__.py]: Adding a docstring here would be nice :-)

File: jupyterlab/__init__.py [hunk 2]
```diff
@@ -8,14 +8,15 @@
 from notebook.base.handlers import IPythonHandler, FileFindHandler
 from jinja2 import FileSystemLoader
 from notebook.utils import url_path_join as ujoin
+from notebook.base.handlers import AuthenticatedHandler
```
Comment (jasongrout) [inline jupyterlab/__init__.py]: We don't need this import anymore.

**Comment by jasongrout** [comment]
The dependencies (such as fullpage, etc.) should be pulled in from npm and imported, not included as source files.

**Comment by jasongrout** [comment]
This is where a template is found. If you want to influence where a template is located, changing this line is a good start: https://github.com/jupyter/jupyterlab/pull/230/files#diff-319b1c023b01f33238afa36efbf90594R48

**Comment by willingc** [comment]
@roshancvp Let's work on rebasing this PR on the morning.

**Pull request closed without merging**